Wikipedia/assets/preview.html

```diff
@@ -12,6 +12,8 @@
       A.edit-page {
         display: none;
       }
+      .section-heading {display: block !important;}
+      .mw-editsection {display: none !important;}
```

Comment:
I couldn't find any references to `.mw-editsection` in the DOM - is it used elsewhere?

Reply:
On edit preview on non-lead section - it shows a giant [Edit] link which we used to hide somehow...